IrisSoftware/widgets.py

```diff
@@ -1,4 +1,6 @@
 """A collection of widgets for the UI."""
+import math
+import sys
 from typing import Any, List, Tuple
```

Comment:
Since we're moving to python 3.9 let's try to be consistent and use list and tuple for type hints (generics in python 3.9) instead of typing List and Tuple!
In this file I can see changes that we can make on lines 279 and 304 
```suggestion
from typing import Any
```